### PR TITLE
Updates for accessor exceptions tests

### PR DESCRIPTION
**Pull request by vasilytric**
Here you can find the following improves:
- Test case names - to improve test case names
- Replace "ifndef" with "ifdef" - to reduce mental load when
  reading the code

### CODE REVIEW

File: tests/accessor/accessor_exceptions_core.cpp [hunk 1]
```diff
@@ -26,21 +26,21 @@ namespace accessor_exceptions_test_core {
 using namespace sycl_cts;
 
 DISABLED_FOR_TEST_CASE(hipSYCL, ComputeCpp, DPCPP)
-("Generic sycl::accessor constructor exceptions test core types.",
+("Generic sycl::accessor constructor exceptions test. core types.",
```
Comment (bader) [inline tests/accessor/accessor_exceptions_core.cpp:29]:
```suggestion
("Generic sycl::accessor constructor exceptions test. Core types.",
```

File: tests/accessor/accessor_exceptions_core.cpp [hunk 2]
```diff
@@ -26,21 +26,21 @@ namespace accessor_exceptions_test_core {
...
-("sycl::local_accessor  constructor exceptions test core types.",
+("sycl::local_accessor  constructor exceptions test. core types.",
  "[accessor]")({
   const auto types = get_conformance_type_pack();
   for_all_types_vectors_marray<run_tests_with_types, local_accessor>(types);
 });
 
 DISABLED_FOR_TEST_CASE(hipSYCL, ComputeCpp, DPCPP)
-("sycl::host_accessor constructor exceptions test core types.", "[accessor]")({
+("sycl::host_accessor constructor exceptions test. core types.", "[accessor]")({
```
Comment (bader) [inline tests/accessor/accessor_exceptions_core.cpp:43]:
```suggestion
("sycl::host_accessor constructor exceptions test. Core types.", "[accessor]")({
```

**Review by bader — APPROVED** [review]

File: tests/accessor/accessor_exceptions_core.cpp (hunk 1 above)
Comment (vasilytric) [reply tests/accessor/accessor_exceptions_core.cpp]: I've applied this comment in his commit: https://github.com/KhronosGroup/SYCL-CTS/pull/359/commits/92f3e15e74282f88fceb1b5c98e796a214ef88a6

File: tests/accessor/accessor_exceptions_core.cpp [hunk 3]
```diff
@@ -26,21 +26,21 @@ namespace accessor_exceptions_test_core {
...
-("Generic sycl::accessor constructor exceptions test core types.",
+("Generic sycl::accessor constructor exceptions test. Core types.",
  "[accessor]")({
   const auto types = get_conformance_type_pack();
   for_all_types_vectors_marray<run_tests_with_types, generic_accessor>(types);
 });
 
 DISABLED_FOR_TEST_CASE(hipSYCL, ComputeCpp, DPCPP)
-("sycl::local_accessor  constructor exceptions test core types.",
+("sycl::local_accessor  constructor exceptions test. Core types.",
```
Comment (bader) [inline tests/accessor/accessor_exceptions_core.cpp:36]:
```suggestion
("sycl::local_accessor constructor exceptions test. Core types.",
```

**Pull request merged**